### PR TITLE
chore: consequent changes for 1-based line indexing

### PR DESCRIPTION
**Pull request by Pranavchiku**
Consequent PR of https://github.com/lfortran/lfortran/pull/5818

### CODE REVIEW

File: server/src/lfortran-accessors.ts [hunk 1]
```diff
@@ -403,11 +405,18 @@ export class LFortranCLIAccessor implements LFortranAccessor {
...
           start.character--;
 
           const end: Position = range.end;
+          end.line--;
           end.character--;
 
+          this.logDebug(
+            "Here URI: %s, range: %s",
+            uri, JSON.stringify(range)
+          );
```
Comment (Pranavchiku) [inline server/src/lfortran-accessors.ts:418]:
```suggestion
```

**Comment by Pranavchiku** [comment]
The checks fail because they are written considering 0 based line indexing. Will push a fix.

**Comment by certik** [comment]
This will be needed too: https://github.com/lfortran/lfortran-lsp/issues/54

**Review by certik — APPROVED** [review]
I am going to merge this to unblock us.

@dylon once you get to reviewing this, if there is an issue, just send a new PR with a fix.

**Pull request merged**